README.md

```diff
@@ -291,6 +292,9 @@ The Expression Language supports the following functions:
   * `input` a timestamp to add to.
   * `delta` a `long` amount of `unit` to add to `input`. Can be a negative value to perform subtraction.
   * `unit` the string unit of time to add or subtract. Can be one of [`years`, `months`, `days`, `hours`, `minutes`, `seconds`, `millis`].
+* `decimal(input, scale)`: Converts `input` to a `BigDecimal` with the given `scale`. 
```

Comment:
Add the possibility to define the precision ?

Reply:
I thought about this, I can add it as optional, but in practice it is only used for Avro's validation and when constructing java's BigDecimal, you can't specify precision - it is effectively the is the number of digits in the unscaled value so with bytes + scale only a user would have full control on their decimal.

Reply:
nm, you're right. Adding precision is useless here